species/read/read_radtrans.py

```diff
@@ -633,6 +633,26 @@ def get_model(
             tau = self.pressure * 1e6 * 10.0 ** model_param["log_delta"]
             temp = (0.75 * model_param["tint"] ** 4.0 * (2.0 / 3.0 + tau)) ** 0.25
 
+        elif (
+            "PTslope_1" in model_param
+        ):
+            num_layer = 6 # could make a variable in the future
+            layer_pt_slopes = np.ones(num_layer) * np.nan
+            for index in range(num_layer):
+                layer_pt_slopes[index] = model_param[f'PTslope_{num_layer - index}']
+
+            try:
+                from petitRADTRANS.physics import dTdP_temperature_profile
+            except ImportError:
```

Comment:
An error would occur in this case below when `dTdP_temperature_profile` is called? Instead (I think) you could perhaps use `raise` within the `except` to create and error with the message from `print`

Reply:
I think I did this except as e, raise statement right?

Reply:
Yes I think what you did is correct. So when it goes to the `except` it will still raise an error